app/views/shared/provider/navigation/service/_integration.html.slim

```diff
@@ -6,6 +6,10 @@
          title: 'Methods & Metrics',
          path: admin_service_metrics_path(@service)
 
+= render vertical_nav_item,
+         title: 'Mapping Rules',
```

Comment:
We should add this new menu item in https://github.com/3scale/porta/blob/73c8e8a616b5367c7b4190624afe8cc93814bfc6/features/old/menu/api_menu.feature#L38-L43

Take a look at https://github.com/3scale/porta/pull/1008/files?file-filters%5B%5D=.feature#diff-311ba69d0277694c725639eaad915d08 in case you need to bypass the rolling update.